### PR TITLE
Dictionary refactoring

### PR DESCRIPTION
**Pull request by raipc**

### CODE REVIEW

**Comment by raipc** [comment]
Depends on https://github.com/axibase/atsd/pull/780

File: chart-of-the-day/hetzner-outage/README.md [hunk 1]
```diff
@@ -32,7 +32,7 @@ Filing ticket for each affected server and watching the Hetzner status page for
 
 ## Time to Resolve
 
-We used the reported counters as inputs to arrive at a rough TTR estimate by dividing the ticket count by the hourly rate, calculated as delta value from the previous report.
+We used the reported counters as inputs to arrive at a rough time-ticket ratio estimate by dividing the ticket count by the hourly rate, calculated as delta value from the previous report.
```
Comment (rodionos) [inline chart-of-the-day/hetzner-outage/README.md]: time to resolution, as in MTTR

File: chart-of-the-day/facebook/README.md [hunk 2]
```diff
@@ -10,15 +10,15 @@ Since launching its IPO in mid-2012 for $38 a share, and being initially valued
 ![](./images/volume-compare.png)
 [![](./images/button.png)](https://apps.axibase.com/chartlab/e2635247#fullscreen)
 
-> The upper chart tracks USD volume of FB common stock, which is skyrocketing in the present week, amid rumors that Mr. Zuckerberg himself is dumping millions of dollars worth of personal stock as well. The lower graph uses a duel [`axis`](https://axibase.com/products/axibase-time-series-database/visualization/widgets/time-chart/#tab-id-2) expression to group unlike data in one widget without hindering visibility.
+> The upper chart tracks USD volume of Facebook common stock, which is skyrocketing in the present week, amid rumors that Mr. Zuckerberg himself is dumping millions of dollars worth of personal stock as well. The lower graph uses a duel [`axis`](https://axibase.com/products/axibase-time-series-database/visualization/widgets/time-chart/#tab-id-2) expression to group unlike data in one widget without hindering visibility.
```
Comment (rodionos) [inline chart-of-the-day/facebook/README.md]: FB is the ticker name for Facebook

File: chart-of-the-day/hetzner-outage/README.md (hunk 1 above)
Comment (kghamilton89) [reply chart-of-the-day/hetzner-outage/README.md]: Corrected.

File: chart-of-the-day/facebook/README.md (hunk 2 above)
Comment (kghamilton89) [reply chart-of-the-day/facebook/README.md]: I left this change as is, but added (`FB`) in parentheses.

**Comment by kghamilton89** [comment]
Links which are read as bad are excepted by this PR: https://github.com/axibase/atsd/pull/781

**Comment by kghamilton89** [comment]
Only remaining errors in Travis build are "dead" links repaired by `atsd#781`.

**Comment by raipc** [comment]
> Looks like the spellchecker was off, or did we accidentally add these type of words to dictionary?

Neither. It was caused by the way yaspeller worked with the dictionary. If the dictionary contained a substring, the spelling error would be ignored. The dictionary generating process change aimed to resolve this issue.

**Pull request merged**